lib/managers/photos_manager.dart

```diff
@@ -32,10 +35,21 @@ abstract class PhotosManagerBase with Store {
   @observable
   ObservableList<int> chosen = ObservableList<int>();
 
-  Iterable<Uint8List> get chosenPhotos => chosen.map((choice) => photos[choice]);
-
+  @observable
   CaptureMode captureMode = CaptureMode.single;
 
+  @computed
+  bool get showLiveViewBackground => photos.isEmpty && captureMode == CaptureMode.single;

```

Comment:
Not sure if a super logical place for this variable, but idk where to put it else either.